src/Database/Database.php

```diff
@@ -114,6 +114,7 @@ public function connect()
 		$pass    = trim($this->configCache->get('database', 'password'));
 		$db      = trim($this->configCache->get('database', 'database'));
 		$charset = trim($this->configCache->get('database', 'charset'));
+                $socket  = trim($this->configCache->get('database', 'socket')); 
```

Comment:
```suggestion
		$socket  = trim($this->configCache->get('database', 'socket')); 
```

Code Standard - Wrong indent